fixtures/coverall/src/coverall.udl

```diff
@@ -22,6 +22,13 @@ dictionary SimpleDict {
     float? maybe_float32;
     double float64;
     double? maybe_float64;
+    Coveralls? coveralls;
+};
+
+[Enum]
+interface MaybeSimpleDict {
+    Yeah(SimpleDict d);
+    Nah();
```

Comment:
This tests codegen for nested object references, I don't have any tests that actually exercise it yet.